src/configurations/oAuthServerSchema.js

```diff
@@ -61,6 +176,29 @@ const oAuthServerSchema = {
             preserve_host: false,
             listen_path: '',
             upstream_url: '',
+            upstreams: {
```

Comment:
If all `upstreams` schemas has the same structure we could declare one and reuse over the code, so we don't repeat

Reply:
totally agree, but here is one **but**:

in future, this schema, probably, will come from the server side...

Reply:
gotcha!